lua/astrocommunity/utility/toggleterm-manager/init.lua

```diff
@@ -0,0 +1,50 @@
+return {
+  {
+    "ryanmsnyder/toggleterm-manager.nvim",
```

Comment:
```suggestion
    "ryanmsnyder/toggleterm-manager.nvim",
    init = function(plugin) require("astrocore").on_load("telescope.nvim", plugin.name) end,
```

load plugin when telescope loads